columns/Column.js

```diff
@@ -41,11 +41,14 @@ export class Column {
      * @param {boolean} [c.resizable] - false to prevent user from drag-and-drop resizing.
      * @param {boolean} [c.movable] - false to prevent user from drag-and-drop re-ordering.
      * @param {Column~rendererFn} [c.renderer] - function to produce a formatted string for each cell.
-     *      Supports HTML as output. Passed both field value and entire row data object.
+     *      Supports HTML as output. Passed the field value as the first parameter, and an object
+     *      containing the record, column, and renderer params (destructured) as the second parameter.
      * @param {function} [c.elementRenderer] - elementFactory function to return a React component
```

Comment:
can we remove the word 'elementFactory' here.  It seems like a loaded term that makes me think of `elemFactory`.
Can just be a "function returning a react component"